app/code/Magento/Catalog/Block/Adminhtml/Product/Edit/Tab/Inventory.php

```diff
@@ -75,6 +82,7 @@ public function __construct(
         $this->coreRegistry = $coreRegistry;
         $this->stockRegistry = $stockRegistry;
         $this->stockConfiguration = $stockConfiguration;
+        $this->json = $json;
```

Comment:
The new constructor parameters should be optional in order to avoid backward compatibility issues.
Please check the [Contribution Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#php) (section *Adding a constructor parameter*) for more details.